src/components/LayoutTree.tsx

```diff
@@ -275,6 +282,13 @@ const LayoutTree = () => {
         }}
       >
         <TransitionGroup>
+          <BulletCursorOverlay
```

Comment:
It does no harm, but it's probably better to move `BulletCursorOverlay` outside `TransitionGroup` for clarity. Only `Transition` and `CSSTransition` components have an effect within `TransitionGroup`. (`TreeNode` returns a `CSSTransition` component.)